src/main/java/org/ibp/api/java/impl/middleware/common/validator/GermplasmValidator.java

```diff
@@ -18,6 +20,9 @@
 	@Autowired
 	private GermplasmDataManager germplasmDataManager;
```

Comment:
I think we can remove GermplasmDataManager usage here by using `GermplasmService#getGermplasmByGIDs`